receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/accumulator.go

```diff
@@ -64,6 +68,7 @@ func (acc *metricDataAccumulator) getMetricsData(containerStatsMap map[string]Co
 	if metadata.Limits.CPU != nil {
 		taskMetrics.CPUReserved = *metadata.Limits.CPU
 	}
+	taskMetrics.CPUUtilized = ((taskMetrics.CPUUsageInVCPU / taskMetrics.CPUReserved) * 100)
```

Comment:
Is it safe to not check CPUReserved == 0 here?

Reply:
For a task, `CPUReserved` cannot be zero. If the value is not present, we calculate it from all the containers. And, in ECS task, you must need to set CPU Limit- either in task level or in container level.

Reply:
Cool - let's add a comment to the effect

Reply:
Added.